docs/layouts/shortcodes/tranparency-reports/tables/2021/eu-requests.html

```diff
@@ -0,0 +1,33 @@
+<table class="tranparency-table">
+  <tr>
+    <th rowspan="2">Requesting authority</th>
+    <th rowspan="2">Number of requests</th>
+    <th colspan="2" scope="colgroup">Content Data Disclosed</th>
+    <th colspan="2" scope="colgroup">Metadata Disclosed</th>
+    <th rowspan="2">Number of affected clients</th>
```

Comment:
```suggestion
    <th rowspan="2">Number of affected customers</th>
```

Reply:
"Clients" is from the original report, so I wouldn't change it. It's also in the tables from 2020.